frontend/src/concepts/pipelines/content/utils.tsx

```diff
@@ -87,6 +91,9 @@ export const getPipelineIdByPipelineVersion = (
   )?.key.id;
 
 export const getPipelineAndVersionDeleteString = (
-  resources: PipelineCoreResourceKF[],
+  resources: PipelineCoreResourceKFv2[],
   type: 'pipeline' | 'version',
 ): string => `${resources.length} ${type}${resources.length !== 1 ? 's' : ''}`;
+
+export const getPipelineResourceUniqueID = (resource: PipelineCoreResourceKFv2): string =>
+  resource.display_name + resource.created_at;
```

Comment:
Why not make this use the id of the resource? Although I don't mind this solution -- you could have done this at the top level -- why did you use index? lol